scripts/validations.py

```diff
@@ -0,0 +1,125 @@
+import itertools
+import json
+import sys
+
+ACTION_CONTEXT = 'context'
+ACTION_COUNT = 'count'
+ACTIONS = (ACTION_CONTEXT, ACTION_COUNT)
+
+UNLISTED_ADDONS = set()
+
+
+def parse_validations(results):
+    return (json.loads(result) for result in results)
+
+
+def unlisted_validations(results, unlisted_addons=None):
```

Comment:
Ah, now I see. It feels weird to be using a global constant this way, though. You generally don't expect them to change at runtime. I'd rather you just pass the parameter and default to an empty set, or use the global directly. Otherwise, at least a comment explaining when/where `UNLISTED_ADDONS` is set would be nice.